chapters/synchronous.tex

```diff
@@ -995,19 +1012,19 @@ \subsection{Sub-clock Partitioning}\label{sub-clock-partitioning}
 \subsection{Sub-clock Inferencing}\label{sub-clock-inferencing}
 
 For each base-clock partition, the base interval needs to be determined and for each sub-clock partition, the sub-sampling factors and shift need to be determined.
-For each sub-clock partition, the interval might be rational or \lstinline!Real! type and known or parametric or being unspecified.
-The sub-clock partition intervals are constrained by \lstinline!subSample! and \lstinline!superSample! factors which might be known (or parametric) or unspecified, as well as by \lstinline!shiftSample!, \lstinline!shiftCounter! and \lstinline!resolution!, or \lstinline!backSample!, \lstinline!backCounter! and \lstinline!resolution!.
+For each sub-clock partition, the interval might be rational or \lstinline!Real! type and known or parameter expression or being unspecified. % This sentence is impossible to parse!
```

Comment:
My preliminary conclusion is that it is plain wrong and should be deleted. It's the base clocks intervals that can be Rational, Real, Boolean, or unspecified - whereas the factors between different sub-clocks must be rational.
However, there might be more - or it might be the case of multiple consistent rational clocks in a base-clock partition that needs to be clarified.

Reply:
Since you mention _two issues_ in the review summary, I conclude that you want this to be deleted.  Regarding the _there might be mode_, I think it's better if you open a separate issue for this, as I wouldn't be the right person to work on this.

Reply:
The sentence has been removed now.  Was that all?

Reply:
Hopefully yes. If someone figures out what it was supposed to mean we can add that back.